tests/unit_tests/db_engine_specs/test_bigquery.py

```diff
@@ -192,3 +192,36 @@ def test_unmask_encrypted_extra() -> None:
             "private_key": "SECRET",
         },
     }
+
+
+def test_unmask_encrypted_extra_when_empty() -> None:
+    """
+    Test that a None value works for ``encrypted_extra``.
+    """
+    from superset.db_engine_specs.bigquery import BigQueryEngineSpec
+
+    old = None
+    new = json.dumps(
+        {
+            "credentials_info": {
+                "project_id": "yellow-unicorn-314419",
+                "private_key": "XXXXXXXXXX",
+            },
+        }
+    )
+
+    assert json.loads(BigQueryEngineSpec.unmask_encrypted_extra(old, new)) == {
```

Comment:
@betodealmeida is this the expected behavior? It doesn't unmask it if the old value is None.